Assignments/Program.cs

```diff
@@ -1 +1,69 @@
-﻿Console.WriteLine("Hello, World!");
+﻿using System;
+using System.Numerics;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T3 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Fibonacci Series Printer:-" + "\x1B[0m");
+      List<int> testCases = new () { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25};
+      foreach (int i in testCases) {
+         Console.Write ($"\nf{i} ==> ");
+         Console.ForegroundColor = Yellow;
+         FiboSeries (i);
+         Console.ResetColor ();
+      }
+      Console.Write ("\n\nTry Yourself! Enter the length of series, l = ");
+      if (int.TryParse (Console.ReadLine (), out int l)) {
+         Console.Write ($"\nf{l} ==> ");
+         Console.ForegroundColor = Yellow;
+         FiboSeries (l);
+         Console.ResetColor ();
+      }
+   }
+
+   /// <summary>Prints the series of n Fibonacci numbers</summary>
+   public static void FiboSeries (int count) {
+      if (count <= 0) Console.WriteLine ("Please enter a +ve value");
+      var (f0, f1) = (0, 1);
+      while (0 < count--) {

```

Comment:
We will use variable at the left of expression and constant value at right of expression. Also, to be clear decrement count inside while loop
```suggestion
      while (count > 0) {
         ...
         count--;

```